extensions/amp-story/1.0/amp-story-viewport-warning-layer.css

```diff
@@ -18,7 +18,7 @@
 
 .i-amphtml-story-no-rotation-overlay {
   position: absolute !important;
-  z-index: 200000!important; /** unsupported-browser-overlay - 1 */
+  z-index: 20000001!important; /** unsupported-browser-overlay - 1 */
```

Comment:
This might need to be `20000000` if it's unsupported-browser-overlay - 1

Reply:
You're totally right, careless copy paste mistake! Thanks!